lib/tasks/max_campaign_goals.rake

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+namespace :campaign_goals do
+  task :max, %i[payment_intent_id campaign_id] => [:environment] do |_task, args|
+    desc 'Max out campaign goals with gift cards'
+    Item.transaction do
+      campaign = Campaign.find(args.campaign_id)
+      payment_intent = PaymentIntent.find(args.payment_intent_id)
+      payment_intent.update!(campaign: campaign)
```

Comment:
Calling out this line that @Jeremy-Xue and I added - the item creator needed the payment intent to have a campaign so that the individual gift cards will be associated with a campaign!